### ISSUE TITLE
Cannot install packages after python 2.7 installation

### ISSUE THREAD

**Issue by seanmcne** [Microsoft/PTVS#2370]
[Window Title]
Visual Studio - Python support

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[V] Show details  [Close]

[Expanded Information]
```
System.InvalidOperationException: Operation cannot be started because the environment is not configured.
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<AbortIfNotReady>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<InstallAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtensionProvider.<InstallPackage>d__38.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtension.<InstallPackage_Executed>d__14.MoveNext()
```

**Comment by huguesv** [comment]
Thanks! I'm closing as a duplicate, this looks identical to this one:
https://github.com/Microsoft/PTVS/issues/2361

**Comment by zooba** [comment]
Despite this looking similar to #2361, I'd appreciate it if you could open Tools, Python, Diagnostic Info and either post the contents as a [gist](https://gist.github.com) and link it here, or email as an attachment to ptvshelp@microsoft.com.

**Comment by seanmcne** [comment]
For sure, I'll email it over in a min.

**Comment by zooba** [comment]
@seanmcne Thanks. Were you installing into pypy for this? If so, the code causing this problem is trying to run `pypy.exe -c "import pip"` and seeing it fail. Any idea why that might be?

**Comment by seanmcne** [comment]
wow that's strange - I was trying to use pypy for a while but ultimately gave up on that.  Not sure why that's running - let me take another look at it here when i get a free moment and ensure that environment config is completely removed.

**Comment by zooba** [comment]
Dup of #2495, fixed in #2502
